proguard/proguard-project.pro

```diff
@@ -0,0 +1,5 @@
+-verbose
+
+# We should at least keep the Exceptions, InnerClasses,
+# and Signature attributes when processing a library.
+-keepattributes Exceptions,InnerClasses,Signature,RuntimeVisibleParameterAnnotations,RuntimeVisibleAnnotations
```

Comment:
Out of curiosity, why do we need these? https://github.com/mapbox/mapbox-navigation-android/pull/2643#discussion_r398245604

Reply:
I believe that we need 
- `-keepattributes` https://www.guardsquare.com/en/products/proguard/manual/usage/attributes 
-  `-verbose` need to have more information during processing